dateparser/custom_lang_detect/__init__.py

```diff
@@ -0,0 +1,19 @@
+import importlib
+
+loaded_parser = None
+language_parser_module = None
+
+def language_parser(paser_module="fast_text"):
+  global loaded_parser
+  global language_parser_module
+
+  try:
+    if loaded_parser == None:
```

Comment:
```suggestion
    if loaded_parser is None:
```

or

```suggestion
    if not loaded_parser:
```

(I think the second is better for this case).